bin/dehydrated

```diff
@@ -0,0 +1,1116 @@
+#!/usr/bin/env bash
```

Comment:
Maybe rather than copying this or pulling it down in the Dockerfile, we could make this a git submodule?